Resources/Prototypes/Objectives/thief.yml

```diff
@@ -205,15 +205,16 @@
     difficulty: 1
 
 - type: entity
-  parent: BaseThiefStealObjective
-  id: ClothingHeadsetAltMedicalStealObjective
+  parent: BaseThiefStealCollectionObjective
+  id: ClothingCommandHeadsetStealObjective #imp edit: i'm branching this out from just the CMO's headset to all command headsets so there is less of a blatant kick-me sign on the CMO
   components:
-  - type: NotJobRequirement
-    job: ChiefMedicalOfficer
   - type: StealCondition
-    stealGroup: ClothingHeadsetAltMedical
+    stealGroup: CommandHeadset
+    minCollectionSize: 4
+    maxCollectionSize: 12
```

Comment:
why does this start and end so high?

Reply:
there are 21 different headsets this applies to so thats where my base numbers were at. if a round has full command then 14 different headsets will be available which makes the maximum high but doable, and if a round has no command then 7 headsets are available which makes the minimum high but doable. i want to encourage ripping the headsets off of peoples ears directly as much as i want them to break into offices and emag lockers

Reply:
this kind of mandates both if you roll high rather than giving a thief options, and it's a lot of secure trespass you have to get away with. we can give it a try, but i feel like this might be reeeaaal hard

Reply:
i was thinking about this while doing errands and realized that having someone take 12 of 14 total command headsets is disruptive in a way that literally every other collection thief objective isnt, because they are a far more limited resource than, say, IDs, so i'm going to drop this to 1-7